db/migrate/20240609061418_tool_details_and_command_removal.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ToolDetailsAndCommandRemoval < ActiveRecord::Migration[7.0]
+  def change
+    add_column :ai_personas, :tool_details, :boolean, default: true, null: false
+    add_column :ai_personas, :tools, :json, null: false, default: []
+    Migration::ColumnDropper.mark_readonly(:ai_personas, :commands)
+
+    execute <<~SQL
+      UPDATE ai_personas
+      SET tools = commands
+    SQL
+  end
```

Comment:
Should we split this into `up` and `down` and make it irreversible?

Reply:
I feel this is a bit of a fantasy on dev side, once we have gone up... the only way to go down is to go up again. it really is only useful in dev

Will sort out the tests though